core/src/mindustry/world/blocks/defense/turrets/PayloadAmmoTurret.java

```diff
@@ -60,8 +60,8 @@ public void init(){
             @Override
             public void build(Building build, Table table){
                 MultiReqImage image = new MultiReqImage();
-                content.blocks().each(i -> filter.get(i) && i.unlockedNow(), block -> image.add(new ReqImage(new Image(block.uiIcon),
-                () -> build instanceof PayloadTurretBuild it && !it.payloads.isEmpty() && it.currentBlock() == block)));
+                content.blocks().each(i -> filter.get(i) && i.unlockedNow(), content -> image.add(new ReqImage(new Image(content.uiIcon),
```

Comment:
This is technically incorrect now, since it only displays blocks, not units. Will fix myself.

Reply:
Yea I noticed it myself and ten didn't do anything about it